ext/bz2/tests/bug72447.phpt

```diff
@@ -8,12 +8,20 @@ $input = "AAAAAAAA";
 $param = array('blocks' => $input);
 
 $fp = fopen('testfile', 'w');
-stream_filter_append($fp, 'bzip2.compress', STREAM_FILTER_WRITE, $param);
+
+
+try {
+    stream_filter_append($fp, 'bzip2.compress', STREAM_FILTER_WRITE, $param);
+} catch (\ValueError $e) {
+    echo $e->getMessage() . \PHP_EOL;
+}
+
 fclose($fp);
 ?>
 --CLEAN--
 <?php
 unlink('testfile');
 ?>
 --EXPECTF--
-Warning: stream_filter_append(): Invalid parameter given for number of blocks to allocate. (0) in %s%ebug72447.php on line %d
+Warning: stream_filter_append(): unable to create or locate filter "bzip2.compress" %s on line %d
```

Comment:
This warning shouldn't be present if an exception has been thrown.

Reply:
ACK